pydriller/domain/commit.py

```diff
@@ -553,6 +553,42 @@ def merge(self) -> bool:
         """
         return len(self._c_object.parents) > 1
 
+    @property
+    def insertions(self) -> int:
+        """
+        Return the insertion lines of the commit.
+
+        :return: int insertion lines
+        """
+        return self._c_object.stats.total["insertions"]
+
+    @property
+    def deletions(self) -> int:
+        """
+        Return the deletion lines of the commit.
+
+        :return: int deletion lines
+        """
+        return self._c_object.stats.total["deletions"]
+
+    @property
+    def lines(self) -> int:
+        """
+        Return the total lines of the commit.
+
+        :return: int insertion + deletion lines
+        """
+        return self._c_object.stats.total["lines"]
+
+    @property
+    def files(self) -> int:
+        """
+        Return the modified files of the commit.
+
+        :return: int modified files number
+        """
+        return len(self._c_object.stats.files)
```

Comment:
```suggestion
        return self._c_object.stats.total["files"]
```

Reply:
According to [this](https://gitpython.readthedocs.io/en/stable/reference.html?highlight=stats.total#git.util.Stats), `total["files"]` should be the number of changed files as int